treeherder/auth/backends.py

```diff
@@ -67,6 +67,19 @@ def _get_id_token_expiry(self, user_info):
         # https://openid.net/specs/openid-connect-core-1_0.html#IDToken
         return user_info['exp']
 
+    def _get_is_staff_from_userinfo(self, user_info):
```

Comment:
this should be named _get_is_sheriff_from_userinfo, as that would be more proper